ES/ESum042.xml

```diff
@@ -644,250 +355,425 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                    <dimensions unit="mm">
+                                       <height>120</height>
+                                       <width>80</width>
+                                       <dim type="intercolumn"/>
+                                    </dimensions>
+                              <dimensions type="margin" corresp="#textarea2">
+                                       <dim type="top">15</dim>
+                                       <dim type="bottom">31</dim>
+                                       <dim type="right">23</dim>
+                                       <dim type="left">10</dim>
+                                       <dim type="intercolumn"/>
+                                    </dimensions>
+                                 <note corresp="#textarea2">All data for <locus target="#127r">127r</locus>.</note>
+                                    <ab type="ruling"/>
+                                     <ab type="pricking"> Primary pricks are partly visible.</ab>
+                                    <ab type="pricking"> Ruling pricks are visible.</ab>
+                              <ab type="ruling" subtype="pattern">1A-1A/0-0/0-0/C</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling.</ab>
+                                    <ab type="ruling"> The bottom line is written above the ruling.</ab>
+                                 </layout>                                 
+                        </layoutDesc>                        
+                     </objectDesc>
+                     
+                     <handDesc>                       
+                        <handNote xml:id="h2" script="Ethiopic" corresp="#p2 #p3_i1.4 #p3_i3 #p3_i4">
+                             <locus from="126r" to="138v"/>
+                            <persName role= "scribe" ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName>                            
+                        <seg type="script">18th-century script.</seg>
+                        <seg type="ink">Black, red.</seg>
+                             <seg type="rubrication"/>
+                             <desc>Careful, but not very regular; by a mediocre scribe. 
+                                Very similar to <ref target="#h1">Hand 1</ref> but 
+                                inferior to it. Rounded letter shapes. <persName role= "scribe" ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName> is mentioned 
+                                as the scribe on <locus target="#138r"/>.</desc>                           
+                           </handNote>
+                     </handDesc>
+                     </physDesc>
+                     
+                     <history>
+                        <origin>
+                           <origDate notBefore="1750" notAfter="1800">Second half of the 18th century</origDate>.
```

Comment:
fullstop should be deleted or moved inside

Reply:
ok